src/main/resources/LoginExamples.csv

```diff
@@ -0,0 +1,6 @@
+tomsmith,SuperSecretPassword!
```

Comment:
you should add one more column in every row - validation message, if it is empty then login credentials are valid!